src/mpyl/utilities/docker/__init__.py

```diff
@@ -92,3 +92,17 @@ def build(logger: Logger, root_path: str, file_path: str, image_tag: str, target
     stream_docker_logging(logger=logger, generator=logs, task_name=f'Build {file_path}:{target}')
     logger.debug(logs)
     return True
+
+
+def get_env_variables(project: Project, run_properties: RunProperties) -> dict[str, str]:
```

Comment:
I think it would be cleaner to narrow the interfaces of the method arguments: pass in the list of properties and the `Target` only.

Reply:
True, we were using more out of `RunProperties` in the old structure